denizen_scripts/HeroCraft/missions/PvE/mission_craft.dsc

```diff
@@ -13,21 +13,79 @@ mission_craft:
     max: mission_craft_complete
   rewards:
     daily: 100
-    weekly: 200
-    monthly: 400
+    weekly: 400
```

Comment:
Rewards are too high, weekly is multiplied by 7, and monthly by 30

Reply:
oh.